tools/attest/README.md

```diff
@@ -46,3 +46,16 @@ Default value is `bin`.
 Path to output file to write attestation report to.
 
 Default is empty, interpreted as stdout.
+
+
+### `verbose`
+
+If set, then logger appends INFO and WARNING logs to Stdout. Default logger has verbosity set to `0`, so verbosity option should be set to appropriate value to append INFO and WARN logs at variable verbosity levels to Stdout.
```

Comment:
Can you replace the first sentence with below:
If set, then the logger can append INFO and WARNING logs to stdout as per the verbosity level.

Reply:
Updated.